Source/GmmLib/CMakeLists.txt

```diff
@@ -146,15 +146,24 @@ MESSAGE("SourceConfiguration:")
 MESSAGE("CommonDir: ${BS_DIR_COMMON}")
 MESSAGE("IncDir: ${BS_DIR_INC}")
 
+# If '-DGMM_KERNEL_5_1_MOCS_TABLE=TRUE' (default is FALSE) passed to cmake
```

Comment:
If '-DGMM_KERNEL_5_1_MOCS_TABLE=TRUE' (default is FALSE) passed to cmake,
gmmlib will use static MOCS table corresponding to Linux kernels >=5.1. Otherwise it
will use MOCS table corresponding to kernels <5.1.

To assure optimal virtualization support, for Gen11+ static MOCS table is actually Cross-OS table.